### PR TITLE
EXPANDR-9605 Update Indicator Extraction mode to All

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
EXPANDR-9605

## Description
Update Indicator Extraction mode to All

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by capanw** [comment]
@johnnywilkes  @BigEasyJ Please review when you guys get a chance

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
Added notes in Jira ticket. TLDR, we should be more specific with extraction (it can be dangerous and resource intensive to extract all)

**Review by johnnywilkes — APPROVED** [review]
@capanw , thanks for making those changes.  Thank you

**Comment by capanw** [comment]
@mmhw  @ssokolovich This is approved from our team. Please review when you can. Thank you.

File: Packs/CortexXpanse/ReleaseNotes/1_0_24.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Incident Types
+##### Xpanse Alert
+Updated the extractSettings mode to All.
```
Comment (ssokolovich) [inline Packs/CortexXpanse/ReleaseNotes/1_0_24.md:3]: Can you please elaborate a bit more on the change? 
I see that you have added indicator extractions to specific fields `"xpanseip` , `"xpansehostname"` , `"xpansedescription"` and not all....
Seems like a mismatch between the change and the RN.

Comment (capanw) [reply Packs/CortexXpanse/ReleaseNotes/1_0_24.md]: @ssokolovich, thanks for pointing it out. I updated the wording to reflect new changes.

**Review by mmhw — APPROVED** [review]

**Pull request merged**